dbc2val/dbcfeeder.py

```diff
@@ -451,6 +452,11 @@ def _get_command_line_args_parser() -> argparse.ArgumentParser:
         action="store_true",
         help="Use SocketCAN (overriding any use of --dumpfile)",
     )
+    parser.add_argument(
```

Comment:
this new parameter should probably also be documented in the README

Reply:
Yes, and we should better discuss if we think this one need to be controllable by environment variable and config file as well, similar to how canport is managed.

```
if args.canport:
        canport = args.canport
    elif os.environ.get("CAN_PORT"):
        canport = os.environ.get("CAN_PORT")
    else:
        canport = config.get(CONFIG_SECTION_CAN, CONFIG_OPTION_PORT, fallback=None)

```